src/django_components/component.py

```diff
@@ -180,14 +181,28 @@ def _get_dir_path_from_component_path(
 class Component(View, metaclass=SimplifiedInterfaceMediaDefiningClass):
     # Either template_name or template must be set on subclass OR subclass must implement get_template() with
     # non-null return.
-    class_hash: ClassVar[int]
+    _class_hash: ClassVar[int]
+
     template_name: ClassVar[Optional[str]] = None
+    """Relative filepath to the Django template associated with this component."""
     template: Optional[str] = None
+    """Inlined Django template associated with this component."""
     js: Optional[str] = None
+    """Inlined JS associated with this component."""
     css: Optional[str] = None
+    """Inlined CSS associated with this component."""
     media: Media
+    """
+    Normalized definition of JS and CSS media files associated with this component.
+
+    NOTE: This field is generated from Component.Media class.
+    """
+    response_class = HttpResponse
```

Comment:
Before, the `render_to_response` always used the `HttpResponse` class. Now users can override this using `response_class`. This was inspired by a mixin class in the same file as `django.views.View`